orchagent/portsorch.cpp

```diff
@@ -1144,6 +1172,11 @@ bool PortsOrch::setPortMtu(sai_object_id_t id, sai_uint32_t mtu)
     /* mtu + 14 + 4 + 4 = 22 bytes */
     attr.value.u32 = (uint32_t)(mtu + sizeof(struct ether_header) + FCS_LEN + VLAN_TAG_LEN);
 
+    if (isMACsecPort(id))
```

Comment:
This is definitely needed for non-gearbox based devices. 
As discussed, do we need this additional 32 bytes to be added for the gbsyncd based devices as well ?

Reply:
Yes, we also needed it for line_side_port for gearbox. @jimmyzhai  will add more on this PR for adapting gearbox mode.

Reply:
Have updated the fix to update port (w/o gearbox) MTU for MACsec.